osu.Game/Overlays/Direct/DownloadButton.cs

```diff
@@ -105,32 +72,11 @@ private void load(OsuColour colours, OsuGame game, BeatmapManager beatmaps)
             };
         }
 
-        private void updateState(DownloadState state)
+        protected override void Dispose(bool isDisposing)
         {
-            switch (state)
-            {
-                case DownloadState.NotDownloaded:
-                    background.FadeColour(colours.Gray4, 500, Easing.InOutExpo);
-                    icon.MoveToX(0, 500, Easing.InOutExpo);
-                    checkmark.ScaleTo(Vector2.Zero, 500, Easing.InOutExpo);
-                    break;
+            base.Dispose(isDisposing);
```

Comment:
any specific reason for this? it should be automatically unbound by the child.

Reply:
oh, i misunderstood bindings then. will remove.